scripts/build-local-image.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+
+IROHA_HOME=$(dirname $(realpath ${BASH_SOURCE[0]}))/..
+MOUNT=/tmp/iroha
+BUILD_PATH=/tmp/build
+PROJ_BIN=/opt/iroha
+docker pull hyperledger/iroha-docker-develop
+# docker build $IROHA_HOME/docker/develop -t iroha-dev
+echo $IROHA_HOME\;$MOUNT\;$BUILD_PATH\;$PROJ_BIN
+# docker run -tdv $IROHA_HOME:$MOUNT hyperledger/iroha-docker-develop "/bin/bash" >/tmp/iroha-build.id
+CONTAINER=$(cat /tmp/iroha-build.id)
+docker exec $CONTAINER bash -c "mkdir -p $BUILD_PATH $PROJ_BIN;
+	cd $BUILD_PATH && cmake -DTESTING=OFF $MOUNT && make;
+	cp $BUILD_PATH/bin/* $PROJ_BIN;
+	ldd $PROJ_BIN/* | egrep -o '\/.*?\.(so(\..*?)?|a)\s' | grep $MOUNT | uniq | xargs cp -t $PROJ_BIN | true;"
```

Comment:
Change to 

```bash
docker exec $CONTAINER sh << COMMANDS
# bash commands here
COMMANDS
```

As [here] (https://github.com/hyperledger/iroha/blob/master/docker/build_in_docker.sh#L22-L51)